src/openfl/text/TextField.hx

```diff
@@ -2953,6 +2953,9 @@ class TextField extends InteractiveObject implements IShaderDrawable {
 					__caretIndex = __text.length;
 					__selectionIndex = 0;
 					
+					__dirty = true;
+					__setRenderDirty ();
```

Comment:
Would it make sense to this instead:

```
__stopCursorTimer ();
__startCursorTimer ();
```

At least this is what we call in all other places, also when the selection changes with `Shfit` or by mouse. You could mabye even replace everything in here by `setSelection(0, __text.length)`?

Reply:
Yeah, I guess that would be better...

Reply:
Done!